test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -223,4 +222,70 @@ describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function() {
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('none');
+    const cantVerifyLink = document.getElementsByClassName('link js-cant-verify');
+    cantVerifyLink[0].click();
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('block');
+    expect(testContext.view.$('.js-help-description')[0].textContent).toMatchInlineSnapshot(
+      '"Are you trying to use a biometric authenticator?Biometric authenticators (fingerprint, face recognition, PIN) will only work on the same device on which they were set up.If available, set up another security method on the device you used to set up your biometric authenticator.Are you trying to use a security key?If you have set up a security key, insert it in a USB port when prompted by the browser and tap on the button or gold disk. Security keys can work on multiple devices."'
+    );
+  });
+
+  it('shows additional text when Can\'t verify? is clicked from Okta_Authenticator', function() {
+    testContext.init = (
+      currentAuthenticator = ChallengeWebauthnResponse.currentAuthenticator.value,
+      authenticatorEnrollments = [],
+      app = { name: 'Okta_Authenticator' },
+    ) => {
+      const appState = new AppState({
+        currentAuthenticator,
+        authenticatorEnrollments,
+        app,
+      });
+      jest.spyOn(appState, 'getRemediationAuthenticationOptions').mockReturnValue(formName => {
+        if (formName === 'select-authenticator-authenticate') {
+          return [{label: 'some authenticator '}];
+        }
+        return [];
+      });
+      jest.spyOn(appState, 'shouldShowSignOutLinkInCurrentForm').mockReturnValue(false);
+      const settings = new Settings({ baseUrl: 'http://localhost:3000' });
+      const currentViewState = {
+        name: 'challenge-authenticator',
+        relatesTo: {
+          value: currentAuthenticator,
+        },
+      };
+      testContext.view = new ChallengeWebauthnView({
+        el: $sandbox,
+        appState,
+        settings,
+        currentViewState,
+      });
+      testContext.view.render();
+    };
+    jest.spyOn(webauthn, 'isNewApiAvailable').mockReturnValue(true);
+    jest.spyOn(BrowserFeatures, 'isSafari').mockReturnValue(false);
+    testContext.init();
+    expect(testContext.view.$('.idx-webauthn-verify-text').text()).toMatchInlineSnapshot(
+      '"You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification."'
+    );
+    expect(testContext.view.$('.js-cant-verify').length).toBe(1);
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('none');
+    const cantVerifyLink = document.getElementsByClassName('link js-cant-verify');
+    cantVerifyLink[0].click();
+    expect(testContext.view.$('.js-help-description').css('display')).toBe('block');
+    expect(testContext.view.$('.js-help-description')[0].textContent).toMatchInlineSnapshot(
```

Comment:
same nit here, either a string or inline snapshot for html. Probably html is better